ios/Classes/Convert.swift

```diff
@@ -325,13 +325,25 @@ class Convert {
         if let draggable = options["draggable"] as? Bool {
             delegate.isDraggable = draggable
         }
+    }
     
-        if let geometry = options["geometry"] as? [[Double]], geometry.count > 0 {
-            var coordinates: [CLLocationCoordinate2D] = []
+    class func getCoordinates(options: Any?)  -> [CLLocationCoordinate2D] {
```

Comment:
rename this to something like `getLineCoordinates` and this LGTM

Reply:
I was wondering if same code can be later reused for other annotations like Circle or Fill. But you're right, in this PR `getLineCoordinates` probably makes more sense

Reply:
hmm it could be indeed be reused for polygons (not for circles or symbols tho) so the name is probably fine!